db/migrate/20181016140921_migrate_orch_stacks_to_have_ownership_concept.rb

```diff
@@ -0,0 +1,83 @@
+class MigrateOrchStacksToHaveOwnershipConcept < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ApplicationRecord
+    self.inheritance_column = :_type_disabled # disable STI
+
+    def tenant_identity
+      User.super_admin
+    end
+  end
+
+  class Service < ApplicationRecord
+    self.inheritance_column = :_type_disabled # disable STI
+    belongs_to :evm_owner, :class_name => "User"
+
+    def tenant_identity
+      user = evm_owner
+      user = User.super_admin if user.nil? || !user.miq_group_ids.include?(miq_group_id)
```

Comment:
Most services are created doing provisioning which would set their owner identify.  They can also be created in automate which allows user/group to be set as well.  This fallback to `User.super_admin` is really just a last ditch effort and not something I would expect to encounter very much.  🎃

Reply:
If you are setting to `super_admin` because the group is off, , can you set the correct `miq_group`?

```suggestion
      if user.nil? || !user.miq_group_ids.include?(miq_group_id)
        user = User.super_admin.tap { |u| u.current_group_id = miq_group_id }
```

Reply:
Ugh, this has been changed to reflect the last comment, it just isn't showing in github's UI as resolved...